app/controllers/site_controller.rb

```diff
@@ -408,6 +408,10 @@ def view_gene_expression
     @cluster_annotations = load_cluster_group_annotations
     @top_plot_partial = @selected_annotation[:type] == 'group' ? 'expression_plots_view' : 'expression_annotation_plots_view'
     @y_axis_title = load_expression_axis_title
+    matrix = @study.expression_matrix_files.first
```

Comment:
What is this local variable `matrix` being used for?  I don't see it accessed anywhere else.

Reply:
Good catch, fixed in fc12bf2.